controls/V-72017.rb

```diff
@@ -1,70 +1,64 @@
 # encoding: utf-8
 #
-=begin
------------------
-Benchmark: Red Hat Enterprise Linux 7 Security Technical Implementation Guide
-Status: Accepted
 
-This Security Technical Implementation Guide is published as a tool to improve
-the security of Department of Defense (DoD) information systems. The
-requirements are derived from the National Institute of Standards and
-Technology (NIST) 800-53 and related documents. Comments or proposed revisions
-to this document should be sent via email to the following address:
-disa.stig_spt@mail.mil.
+EXEMPT_HOME_USERS = attribute(
```

Comment:
These blocks are all repeated and should probably be abstracted into a set of helper methods in `libraries`